### DISCUSSION TITLE
Hi did not find relevant answer in docs [Apr 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5567] [Discord Threads]
> 🙋‍♂️ **elephantum**

**Comment by o1lab** [comment]
> 🙋‍♂️ **elephantum**  
#### Looking into code it seems like it's not possible 😦  
#  
> 🙋‍♂️ **elephantum**  
#### It seems like all that's needed is to provide `endpoint` to `AWS.S3` constructor if something like `NC_S3_ENDPOINT` is set.   
####    
#### https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#constructor-property  
#  
> 🙋‍♂️ **elephantum**  
#### Somewhere here: https://github.com/nocodb/nocodb/blob/develop/packages/nocodb/src/lib/plugins/s3/S3.ts#L107  
#  
> 💁 **o1lab**  
#### what s3 service are you using ? if its minio or similar  
#  
> 💁 **o1lab**  
#### you shud be able to configure  
#  
> 💁 **o1lab**  
#### from our 'app' store  
#  
> 💁 **o1lab**  
#### https://docs.nocodb.com/setup-and-usages/account-settings#app-store  
#  
> 🙋‍♂️ **elephantum**  
#### <@310157751395287040> nice, looks like something that will work.   
####    
#### Is it possible to provide some sort of config file instead of manual configuration?  
#  
> 🙋‍♂️ **elephantum**  
#### Minio connector works for me! Even though my cloud provider has it's own (non-minio) S3-compatible implementation.   
####    
#### If it would be possible to setup minio connector from app store with config file - it would be awesome!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1088911990837215272)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
